### ISSUE TITLE
Operator for launching in parallel several tasks.

### ISSUE THREAD

**Issue by DavidPerezIngeniero** [sbt/sbt#2016]
Like we have ";" for concatenating tasks sequentially, we could also have another symbol for parallel execution.

This would allow to launch several tasks in parallel from the command line.

More details in this [Stackoverflow question](http://stackoverflow.com/questions/30185597/how-to-parallellize-tasks-given-on-the-command-line/30186073?noredirect=1#comment48492570_30186073).

**Comment by jsuereth** [comment]
You should use the `all` command.   Closing this in favor of stackoverflow answers.

**Comment by dwijnand** [comment]
Ah I didn't know about `all`. Thanks Josh and sorry for the noise.

**Comment by DavidPerezIngeniero** [comment]
Thanks jsuereth!.
